code/modules/power/singularity/field_generator.dm

```diff
@@ -42,26 +42,27 @@ field_generator power level display
 	var/list/obj/machinery/field/containment/fields
 	var/list/obj/machinery/field/generator/connected_gens
 	var/clean_up = 0
-
-/obj/machinery/field/generator/update_icon()
-	cut_overlays()
-	if(warming_up)
-		add_overlay("+a[warming_up]")
-	if(fields.len)
-		add_overlay("+on")
-	if(power_level)
-		add_overlay("+p[power_level]")
-
+	var/static/loose_message_cooldown = 0 //Cooldown var here as a static var so its shared between all field generators
```

Comment:
Could use defines
If the define for static cooldown doesn't exist adding it wouldn't be much of a pain